tsconfig.json

```diff
@@ -1,16 +1,15 @@
 {
+    "extends": "@tsconfig/node18-strictest/tsconfig.json",
     "compilerOptions": {
-        "lib": ["ES2020"],
         "noImplicitAny": true,
         "removeComments": true,
-        "moduleResolution": "node",
-        "esModuleInterop": true,
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
         "sourceMap": true,
-        "target": "ES2020",
+        "noImplicitReturns":false,
```

Comment:
Not sure I understand the reasoning for disabling this?

Reply:
An example for this is when you have a try/catch block and you do not want to return a value when an error occurs. With `noImplicitReturns` set to `true`, you will see the error/warning as shown below. To suppress this error, you have to return something inside catch block.
![image](https://user-images.githubusercontent.com/108910975/236385138-1eb79c0a-1b24-4b04-af2a-a7641425a1cd.png)

Reply:
I was confused by the double negative here but agree with @TheOrangePuff , this seems like a rule we want enabled.

Reply:
I disabled this just to avoid having return everywhere (even when we don't need it) as we tend to only return where we need to return something. However, I don't have strong opinion on this.